usermanagerlib/src/test/java/org/pesmypetcare/usermanagerlib/clients/PetManagerClientTest.java

```diff
@@ -83,6 +92,51 @@ public void setUp() {
             + "    \"recommendedKcal\": \"2.5\"\n"
             + "  }\n"
             + "}]");
+        recKcalJson = new StringBuilder("{\n"
+            + "  \"recommendedKcal\": 2.5\n"
+            + "}");
+        needsJson = new StringBuilder("{\n"
+            + "  \"needs\": \"None of your business\"\n"
+            + "}");
+        mealsFieldCollectionJson = new StringBuilder(
+            "[\n" +
+                "  {\n" +
+                "    \"body\": {\n" +
+            "      \"kcal\": 85.44,\n" +
+            "      \"mealName\": \"Tortilla\"\n" +
+            "    },\n" +
+            "    \"key\": \"1990-01-08T15:20:30\"\n" +
+            "  }\n" +
+            "  ,{\n" +
+            "    \"body\": {\n" +
+            "      \"kcal\": 85.44,\n" +
+            "      \"mealName\": \"Tortilla\"\n" +
+            "    },\n" +
+            "    \"key\": \"1995-01-08T15:20:30\"\n" +
+            "  }\n" +
+            "  ,{\n" +
+            "    \"body\": {\n" +
+            "      \"kcal\": 85.44,\n" +
+            "      \"mealName\": \"Tortilla\"\n" +
+            "    },\n" +
+            "    \"key\": \"1998-01-08T15:20:30\"\n" +
+            "  }\n" +
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['+' should be on a new line.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5479466)